Client/Frontend/Home/TopSites/TopSitesDimension.swift

```diff
@@ -77,14 +77,17 @@ class TopSitesDimensionImplementation: TopSitesDimension {
         return numberOfRows - Int(numberOfEmptyCellRows.rounded(.down))
     }
 
+    //Ecosia: The design decided to have 4 tiles per row by default.
+    //Commenting the logic based on the interface and return 4.
     /// Get the number of tiles per row the user will see. This depends on the UI interface the user has.
     /// - Parameter interface: Tile number is based on layout, this param contains the parameters needed to computer the tile number
     /// - Returns: The number of tiles per row the user will see
     private func getNumberOfTilesPerRow(for interface: TopSitesUIInterface) -> Int {
-        if interface.horizontalSizeClass == .regular {
-            return 6
-        } else {
-            return 4
-        }
+        // Ecosia if interface.horizontalSizeClass == .regular {
```

Comment:
not a blocker but for multiline comments we tend to use block comments /* comment */ as they are less invasive regarding the resulting change set. 

Anyhow especially here the code was modified by us already. So I think you can completely remove the logic and just return 4. No need for comments.

Reply:
Oh, nice 🙌 - will modify it straightaway, returning just `4`👌.
Regarding the multiline, I'm absolutely on the same page. I've copied the format from another comment